widget/view.controller.js

```diff
@@ -2,38 +2,186 @@
   MIT License
   Copyright (c) 2024 Fortinet Inc
   Copyright end */
-  
+
 'use strict';
 (function () {
-    angular
-      .module('cybersponse')
-      .controller('soarFrameworkConfigurationWizard100Ctrl', soarFrameworkConfigurationWizard100Ctrl);
-
-    soarFrameworkConfigurationWizard100Ctrl.$inject = ['$scope', 'widgetUtilityService', '$rootScope', 'widgetBasePath', 'WizardHandler'];
-
-    function soarFrameworkConfigurationWizard100Ctrl($scope, widgetUtilityService, $rootScope, widgetBasePath, WizardHandler) {
-      $scope.moveNext = moveNext;
-      $scope.isLightTheme = $rootScope.theme.id === 'light';
-      $scope.startPageImage = $scope.isLightTheme ? widgetBasePath +'images/sfspStartLight.png': widgetBasePath +'images/sfspStartDark.png';
-
-      function _handleTranslations() {
-        widgetUtilityService.checkTranslationMode($scope.$parent.model.type).then(function () {
-          $scope.viewWidgetVars = {
-            // Create your translating static string variables here
-          };
-        });
+  angular
+    .module('cybersponse')
+    .controller('soarFrameworkConfigurationWizard100Ctrl', soarFrameworkConfigurationWizard100Ctrl);
+
+  soarFrameworkConfigurationWizard100Ctrl.$inject = ['$scope', 'widgetUtilityService', '$rootScope', 'widgetBasePath', 'WizardHandler', 'soarConfigService', 'toaster'];
+
+  function soarFrameworkConfigurationWizard100Ctrl($scope, widgetUtilityService, $rootScope, widgetBasePath, WizardHandler, soarConfigService, toaster) {
+    $scope.gblVarToKeyStoreMapping = {
```

Comment:
Also instead of making a separate record everything can be accumulated into a single JSON record